scripts/build

```
@@ -209,11 +209,10 @@ for p in $_pkg_depends; do
 done
 
 if [ "${BUILD_WITH_DEBUG}" = "yes" ]; then
-  printf "%${BUILD_INDENT}c $(print_color CLR_BUILD "BUILD")    $PACKAGE_NAME $(print_color CLR_TARGET "($TARGET)") [DEBUG]\n" ' '>&$SILENT_OUT
+  build_msg "CLR_BUILD" "BUILD" "${PACKAGE_NAME} $(print_color CLR_TARGET "(${TARGET})" [DEBUG])" "indent"
```

` [DEBUG]` is not a parameter to `print_color()` - this line should be:
```suggestion
  build_msg "CLR_BUILD" "BUILD" "${PACKAGE_NAME} $(print_color CLR_TARGET "(${TARGET})") [DEBUG]" "indent"
```